docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -95,45 +78,45 @@ The main features of the Autofill Manager API include:
    }
    ```
 
-   You can use the autofill manager functions after receiving `AUTOFILL_MANAGER_CONNECTION_STATUS_CONNECTED` in `manager_connection_status_changed_cb()`.
+   You can use the Autofill manager functions after receiving `AUTOFILL_MANAGER_CONNECTION_STATUS_CONNECTED` in `manager_connection_status_changed_cb()`.
 
-## Getting installed autofill service list
+## Get Installed Autofill Service List
 
 To get the installed autofill service list:
 
 1. To access Autofill service information, create a callback function using the `autofill_service_info_cb()` function, which is invoked for each installed Autofill service application:
```

Comment:
Please can the sentence be rephrased to:
Create a callback function using the `autofill_service_info_cb()` function, which is invoked for each installed autofill service application and used to access Autofill service information.

Reply:
fixed